deploy.sh

```diff
@@ -25,11 +25,11 @@ push_helm_charts() {
 }
 
 if [ "${TRAVIS_BRANCH}" == 'csi-v0.3' ]; then
-	export RBD_IMAGE_VERSION='v0.3.0'
-	export CEPHFS_IMAGE_VERSION='v0.3.0'
+	export ENV_RBD_IMAGE_VERSION='v0.3.0'
+	export ENV_CEPHFS_IMAGE_VERSION='v0.3.0'
 elif [ "${TRAVIS_BRANCH}" == 'csi-v1.0' ]; then
-	export RBD_IMAGE_VERSION='v1.0.0'
-	export CEPHFS_IMAGE_VERSION='v1.0.0'
+	export ENV_RBD_IMAGE_VERSION='v1.0.1-canary'
```

Comment:
AFAIK we agreed to have only `canary` images tag for csi-v1.0 and also please send a PR to master branch

Reply:
We need to stop publishing images from 1.0 branch, what is the strategy for doing that?

Also please update these issues with the same, for users on these branches, #296 #345 

The intention here is to have rolling images, such that users can stick to a particular version of the image rather than have the same image updated.

Your strategy seems to be to call master 1.0 and release canary from the same. I think we may need to inform users of this change and proceed, at present there is confusion on the messaging.